x/mint/simulation/genesis.go

```diff
@@ -53,15 +56,42 @@ func GenDistributionProportions(r *rand.Rand) types.DistributionProportions {
 	staking := r.Int63n(99)
 	left := int64(100) - staking
 	incentives := r.Int63n(left)
-	communityPool := left - incentives
+	left -= incentives
+	devs := r.Int63n(left)
+	communityPool := left - devs
```

Comment:
```suggestion
	var (
		staking		= r.Int63n(99)
		left		= int64(100) - staking
		incentives	= r.Int63n(left)
		left 		= incentives - 1
		devs		= r.Int63n(left)
		communityPool	= left - devs
	)
```

Reply:
can't really do this, `left` is used to track the different allocations and updated as they are generated. They can't be grouped in `var`